osafw-app/App_Code/models/DBUpdates.cs

```diff
@@ -0,0 +1,65 @@
+// DBUpdates model class
+//
+// Part of ASP.NET osa framework  www.osalabs.com/osafw/asp.net
+// (c) 2009-2023 Oleg Savchuk www.osalabs.com
+
+using System.Collections;
+using System.ComponentModel.DataAnnotations;
+using System.Reflection;
+
+namespace osafw;
+
+public class DBUpdates : FwModel
+{
+    public const int STATUS_APPLIED = 50;
+    public DBUpdates() : base()
+    {
+        db_config = "";
+        table_name = "db_updates";
+        
+    }
+
+    public void parseUpdates()
```

Comment:
Well, what if non-applied sql file content changed (i.e. it was still in development or some typo)?
I don't like storing sql in idesc.

My initial idea was to only store in db applied updates
If you want to import files into db, just import filenames, and use sql from the file. This will avoid content duplication/synchronization.